src/libsync/theme.h

```diff
@@ -408,15 +417,15 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      * important dependency versions.
      */
     virtual QString versionSwitchOutput() const;
-	
-	/**
```

Comment:
I'm unsure if this changes were necessary? Maybe there was some auto-formating going on?